Common/src/main/java/dk/gruppeseks/bodtrd/common/data/entityelements/Path.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package dk.gruppeseks.bodtrd.common.data.entityelements;
+
+/**
+ *
+ * @author Frederik
+ */
+public class Path
+{
+    private Position[] _path;
```

Comment:
overall i like the idea using an array to back this class. however i think it would make more sense to implement the methods like you would in a stack.